src/js/apps/patients/sidebar/action-sidebar_app.js

```diff
@@ -198,13 +198,13 @@ export default App.extend(extend({
     });
   },
   onPostNewComment({ model }) {
-    model.set({ created_at: dayjs.utc().format() }).save();
+    model.set({ created_at: dayjs.utc().format() }).save().then(comment => {
+      Radio.request('ws', 'add', comment);
+    });
```

Comment:
Not sure if we want or need to test this change specifically... the api requests in cypress are almost instant so we'd have to add a delay or something to the request.